src/main/java/seedu/manager/event/EventList.java

```diff
@@ -30,6 +30,9 @@ public EventList(){
      * @return The list of events.
      */
     public ArrayList<Event> getList() {
+        for (Event event : eventList) {
+            System.out.println(event);
+        }
         return eventList;
     }
```

Comment:
I think this method could be changed to return a string with the entire event list, rather than printing it out directly?

Reply:
This was just a temporary function as the list function was not done at this point in time. It will be removed in this merge.